app/[locale]/page.tsx

```diff
@@ -479,6 +465,74 @@ const Page = async ({ params }: { params: PageParams }) => {
             )}
           </div>
 
+          <hr className="!-mb-8 !mt-0 border-disabled lg:!-mb-12" />
```

Comment:
Just noting the coloring here is different than on other pages

<img width="1800" height="425" alt="image" src="https://github.com/user-attachments/assets/8884178f-dae0-4b77-8f9f-1c5da63642aa" />

vs

<img width="1777" height="813" alt="image" src="https://github.com/user-attachments/assets/5a947492-28f9-4b98-b780-92783f4b6fd8" />

Also, we have a `ui/divider` component, but it looks different as well

<img width="1712" height="355" alt="image" src="https://github.com/user-attachments/assets/69faa0c5-5f7d-4a90-9f56-d6a4aeeea69f" />

cc: @konopkja